getting-started/kubernetes/self-managed-onprem/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Self-managed on-premises
-description: Install Calico networking and network policy for on-premises deployments. 
+description: Managing Calico on-premises provides the most flexibility and customization, but requires more resources and infrastructure.  
```

Comment:
This change seems to go in the opposite direction to the others.  i.e. It takes it away from telling you what this section covers, and introduces doubt as to whether you should run Calico on-prem.  I suggest reverting to the previous text for this one.

Reply:
But the original text (Install Calico networking and network policy for on-premises deployments), was the same for the topic as the landing page. I understood you did not want it repeated, so I tried to come up with something different.

Reply:
Must have been a miscommunication on my part, but alas I don't recall what I may have meant to say. I suggest reverting to the previous text for this one.